### PR TITLE
Restart the groomer if it gets stale

### PR DESCRIPTION
**Pull request by obino**
This pull makes sure we have a consistent way to stop running application via monit, and in particular for groomer that we have check on memory and staleness on the daemon.

### CODE REVIEW

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/615/

File: AppController/lib/backup_restore_service.rb [hunk 1]
```diff
@@ -14,8 +14,10 @@ module BackupRecoveryService
   # Starts the BR Service on this machine. We don't want to monitor
   # it ourselves, so just tell monit to start it and watch it.
   def self.start()
-    start_cmd = "/usr/bin/python /root/appscale/AppDB/backup/backup_recovery_service.py"
-    stop_cmd = "/usr/bin/pkill -9 backup_recovery_service"
+    bk_service = self.scriptname()
+    start_cmd = "/usr/bin/python2 #{bk_service}"
```
Comment (menivaitsi) [inline AppController/lib/backup_restore_service.rb]: Use the PYTHON27 constant here and below.

File: AppController/lib/backup_restore_service.rb [hunk 2]
```diff
@@ -14,8 +14,10 @@ module BackupRecoveryService
   # Starts the BR Service on this machine. We don't want to monitor
   # it ourselves, so just tell monit to start it and watch it.
   def self.start()
-    start_cmd = "/usr/bin/python /root/appscale/AppDB/backup/backup_recovery_service.py"
-    stop_cmd = "/usr/bin/pkill -9 backup_recovery_service"
+    bk_service = self.scriptname()
```
Comment (menivaitsi) [inline AppController/lib/backup_restore_service.rb]: Call this variable br_script for consistency.

Comment (menivaitsi) [inline AppController/lib/backup_restore_service.rb]: Never mind, different file.

**Pull request merged**